packages/path_provider/path_provider_platform_interface/pubspec.yaml

```diff
@@ -3,21 +3,20 @@ description: A common platform interface for the path_provider plugin.
 homepage: https://github.com/flutter/plugins/tree/master/packages/path_provider/path_provider_platform_interface
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 1.0.3
+version: 2.0.0-nullsafety
 
 dependencies:
   flutter:
     sdk: flutter
-  meta: ^1.0.5
+  meta: ^1.3.0-nullsafety.3
   platform: ">=2.0.0 <4.0.0"
-  plugin_platform_interface: ^1.0.1
+  plugin_platform_interface: 1.1.0-nullsafety
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  pedantic: ^1.8.0
-  test: any
+  pedantic: ^1.10.0-nullsafety.1
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
-  flutter: ">=1.10.0 <2.0.0"
+  sdk: '>=2.10.0-56.0.dev <3.0.0'
```

Comment:
This needs to be at least `2.12.0` https://github.com/flutter/flutter/issues/69771 . (I'm going to update the plugins already migrated)

Reply:
Done.